projects/schematics/src/migrations/3_0/methods-and-properties-deprecations/data/checkout-group.actions.migration.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  CHECKOUT_ACTIONS,
+  PLACE_ORDER_ACTION,
+  SPARTACUS_CORE,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CHECKOUT_GROUP_ACTIONS_MIGRATIONS: MethodPropertyDeprecation[] = [
```

Comment:
As Patrick stated in my PR, lets stay consistent and change the variable to `CHECKOUT_ACTIONS_MIGRATION` (notice removal of s)

Reply:
also the need to add the path to file so in this case would be to the index I'm assuming